src/Game/Game.cpp

```diff
@@ -23,6 +23,26 @@
 #include "../UI/FpsCounter.h"
 #include "../UI/TextArea.h"
 
+namespace
+{
+    Falltergeist::Graphics::RendererConfig cfg_from_settings(const Falltergeist::Settings& settings)
+    {
+        Falltergeist::Graphics::RendererConfig cfg{
```

Comment:
Can you please use full name? Like `config` instead of `cfg`. Otherwise it affects readability of the code